bundles/erp-order-page-search/src/FondOfOryx/Client/ErpOrderPageSearch/Plugin/SearchExtension/ToDateErpOrderPageSearchQueryExpanderPlugin.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace FondOfOryx\Client\ErpOrderPageSearch\Plugin\SearchExtension;
+
+use DateTime;
+use Elastica\Query;
+use Elastica\Query\BoolQuery;
+use Elastica\Query\Range;
+use FondOfOryx\Shared\ErpOrderPageSearch\ErpOrderPageSearchConstants;
+use Generated\Shared\Search\ErpOrderIndexMap;
+use InvalidArgumentException;
+use Spryker\Client\Kernel\AbstractPlugin;
+use Spryker\Client\SearchExtension\Dependency\Plugin\QueryExpanderPluginInterface;
+use Spryker\Client\SearchExtension\Dependency\Plugin\QueryInterface;
+
+class ToDateErpOrderPageSearchQueryExpanderPlugin extends AbstractPlugin implements QueryExpanderPluginInterface
```

Comment:
I think `ToErpOrderPageSearchQueryExpanderPlugin` will be a better name. It is shorter.